streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -274,6 +289,71 @@ public void setUncaughtExceptionHandler(final Thread.UncaughtExceptionHandler eh
             thread.setUncaughtExceptionHandler(eh);
     }
 
+
+    /**
+     * Find all of the instances of {@link StreamsMetadata} in a {@link KafkaStreams application}
```

Comment:
nit: we can omit "in a {@link KafkaStreams} application", and ditto in the return clause I think.